src/components/Api.js

```diff
@@ -0,0 +1,145 @@
+export default class Api {
+  constructor(options) {
+    this.baseUrl = options.baseUrl;
+    this.headers = options.headers;
+  }
+  getInitialCards() {
+    console.log("Fetching initial cards");
+    return fetch("https://around-api.en.tripleten-services.com/v1/cards", {
+      headers: {
+        Authorization: "89dc4b2f-fab0-42f3-ad8c-2593f7f5189c",
+      },
+    })
+      .then((res) => {
+        if (res.ok) {
+          return res.json();
+        } else {
+          return Promise.reject(`Error: ${res.status}`);
+        }
+      })
+      .then((data) => {
+        console.log(data);
+        return data;
+      })
+      .catch((error) => {
+        console.error(error);
```

Comment:
You should **not** use `catch` in `Api` because you need to `catch` possible errors at the very **end** of any request, and this end is in `index.js`   after all `then` blocks. And in real projects the `catch` blocks usually open error popups and so on.